qutebrowser/qutebrowser.py

```diff
@@ -63,6 +63,7 @@ def get_argparser():
                                      description=qutebrowser.__description__)
     parser.add_argument('-B', '--basedir', help="Base directory for all "
                         "storage.")
+    parser.add_argument('-C', '--config', help="Path to config.py")
```

Comment:
I'd prefer to call it `--config-py` so it's clear that it's only a path to a `config.py` file, not a config folder.